src/canvas.typ

```diff
@@ -46,7 +46,7 @@
     transform:
       ((1, 0,-.5, 0),
        (0,-1,+.5, 0),
-       (0, 0, .0, 0),
+       (0, 0,  1, 0),
```

Comment:
Idk why there was a zero. And the `inverse` function should normally fail to compute an inverse of that matrix.